examples/local/101_initial_cluster.sh

```diff
@@ -38,7 +39,7 @@ for i in 100 101 102; do
 done
 
 # set one of the replicas to primary
-vtctldclient PlannedReparentShard commerce/0 --new-primary zone1-100
+vtctldclient PlannedReparentShard commerce/0 --new-primary zone1-100 --server ${hostname}:${vtctld_grpc_port}
```

Comment:
As @doeg pointed out, I think this is needed because of https://github.com/vitessio/vitess/pull/9579/files

Reply:
No this change isn't required, since in the beginning of the file we call `source ./env.sh` and it aliases `vtctldclient` -
```
alias vtctldclient="command vtctldclient --server localhost:15999"
```

Reply:
Ah, I'll remove this.